src/tests/utils/database-helpers.ts

```diff
@@ -132,9 +133,11 @@ export default class DatabaseHelpers {
    * If editorTools is not passed, it's value in database would be []
    */
   public async insertUser(user?: UserMockCreationAttributes): Promise<User> {
+    const randomPartSize = 6;
+    const randomPart = nanoid(randomPartSize);
     const editorTools = user?.editorTools ?? '[]';
-    const name = user?.name ?? 'CodeX';
-    const email = user?.email ?? 'test@codexmail.com';
+    const name = user?.name ?? `CodeX-${randomPart}`;
```

Comment:
What is `randomPart`? For what?

Reply:
to randomize user's creation, so now if you create more than one user in test, you don't have to explicitly give them different emails and names. (Email and name have unique constraint in db)